### PR TITLE
Improve tooltip consistency

### LINKED ISSUE #5228: UI: Menu icons hover text inconsistency
### Bug Description

Not all menu icons show hover text. These icons can be confusing.

### How to Reproduce

Try hovering over all the menu icons

### Expected Behavior

It should show some hover text similar to this:

Screenshot 2023-08-02 at 10 36 12 AM

### Operating system(s) on local side and remote side

Macos

### RustDesk Version(s) on local side and remote side

1.2.2 (nightly)

### Screenshots

Screenshot 2023-08-02 at 10 36 02 AM

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by beelchester**
Fixes #5228 
Enabled tooltip for menu buttons
Added global tooltip wait duration to 1.5 sec as per https://github.com/rustdesk/rustdesk/issues/5228#issuecomment-1672612609

### CODE REVIEW

File: flutter/lib/desktop/widgets/remote_toolbar.dart [hunk 1]
```diff
@@ -2003,6 +2009,7 @@ class _MultiMonitorMenu extends StatelessWidget {
         Obx(() {
           RxInt display = CurrentDisplayState.find(id);
           return _IconMenuButton(
+            tooltip: "Multi monitor",
```
Comment (rustdesk) [inline flutter/lib/desktop/widgets/remote_toolbar.dart]: this tooltip can tell user nothing. Meaningless tooltip is worse than no tooltip.

Comment (rustdesk) [reply flutter/lib/desktop/widgets/remote_toolbar.dart]: you should give a really helpful tooltip per context. if can not, we should remove it, not only for  consistency.

Comment (beelchester) [reply flutter/lib/desktop/widgets/remote_toolbar.dart]: Ok, I will remove that one.

**Comment by rustdesk** [comment]
Tooltips missed here also.
image

**Comment by rustdesk** [comment]
image

**Comment by sahilyeole** [comment]
My bad, I will add that.

**Comment by sahilyeole** [comment]
> Tooltips missed here also. image

I'm not sure what tooltip message should I give for the tag(#) icon.
Can you suggest something?

**Comment by rustdesk** [comment]
Clicking on it? you can know what does it do

**Comment by rustdesk** [comment]
Show / hide tags, toggle tags, or some other.

**Pull request merged**